### PR TITLE
PYTHON-3383 [pymongoexplain] Support Python 3.7-3.12

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (ShaneHarvey) [inline .github/CODEOWNERS:2]: I see this error:
> Unknown owner on line 2: make sure the team @mongodb/dbx-python exists, is publicly visible, and has write access to the repository

Maybe it's the "write access" part?

Comment (blink1073) [reply .github/CODEOWNERS:2]: Oh, this is a different org, yeah...

**Review by ShaneHarvey — APPROVED** [review]

File: .github/CODEOWNERS
Comment (blink1073) [reply .github/CODEOWNERS:2]: I'm waiting for the access to be granted before merging.

**Pull request merged**
